README.md

```diff
@@ -1 +1,50 @@
-# OOP_School_Library
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+# OOP-school-library
+
+> This is a an app to record books in the library, display the name of the broower. It also store record of student, person, and teacher.
+
+## Built With
+
+- ![](https://img.shields.io/badge/Github-blueviolet)
+- ![](https://img.shields.io/badge/Ruby-red)
+
+- Gitflow
+- Linters (Lighthouse, Webhint, Stylelint, Eslint)
+- Visual Studio Code
+- Git & Github
+
+## Author
+
+👤 **Henry Varflay Singbeh**
+
+- GitHub: [@githubhandle](https://github.com/henrycode460)
+- Twitter: [@twitterhandle](https://twitter.com/460code)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- git
+- Broswer to Display
+- IDE to run and edit the code
+
+### Setup
+
+- open a terminal
+- run ``` git clone git@github.com:henrycode460/OOP_School_Library.git```
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/henrycode460/OOP_School_Library/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The link of the MIT is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯